modules/preferences/main/index.coffee

```diff
@@ -143,6 +143,14 @@ LocalStoragePreferencesStore = {
 
 lookupLocale = (locale) -> localeList.filter((l) -> l.value.toLowerCase() is locale.toLowerCase())[0]
 
+zeroPad = hx.format.zeroPad(2)
+defaultTimezoneLookup = (offset) ->
+  modifier = if offset > 1 then '-' else '+'
```

Comment:
If `offset` is 1 then you'd get 'UTC+00:01', but if `offset` is 2 you would have 'UTC-00:02'. I'm not sure which of the two is correct, but I'd hazard a guess that they should have the same sign :-p